spec/requests/user/user_show_spec.rb

```diff
@@ -1,52 +1,71 @@
 require "rails_helper"
 
 RSpec.describe "UserShow", type: :request do
-  let_it_be(:user) { create(:user, email_public: true, currently_hacking_on: "JSON-LD", education: "DEV University") }
+  let_it_be(:user) { create(:user, email_public: true, education: "DEV University") }
+  let(:doc) { Nokogiri::HTML(response.body) }
+  let(:text) { doc.at('script[type="application/ld+json"]').text }
+  let(:response_json) { JSON.parse(text) }
+  let(:main_entity_of_page) { { "@type" => "WebPage", "@id" => URL.user(user) } }
+  let(:same_as) do
+    [
+      "https://twitter.com/#{user.twitter_username}",
+      "https://github.com/#{user.github_username}",
+      user.mastodon_url,
+      user.facebook_url,
+      user.youtube_url,
+      user.linkedin_url,
+      user.behance_url,
+      user.stackoverflow_url,
+      user.dribbble_url,
+      user.medium_url,
+      user.gitlab_url,
+      user.instagram_url,
+      user.twitch_username,
+      user.website_url,
+    ]
+  end
+
+  let(:disambiguating_description) do
+    [
+      user.mostly_work_with,
+      user.currently_hacking_on,
+      user.currently_learning,
+    ]
+  end
+
+  let(:works_for) { [{ "@type" => "Organization", "name" => user.employer_name, "url" => user.employer_url }] }
```

Comment:
Having these single-use `let` is a bad practice. You might be doing this because of rubocop rule for too many line. I suggest you disable that rule inline and keep these variable within the spec on line 53

Reply:
That's good to know! I definitely do not like having it this way, but I was running into an `ExampleLength` Rubocop error, so I extracted it out to satisfy that. I'll make the change to disable it instead then. 😊